src/common/util/submit-data.js

```diff
@@ -78,15 +34,24 @@ submitData.xhr = function xhr ({ url, body = null, sync, method = 'POST', header
 }
 
 /**
- * Send by appending an <img> element to the page. Do NOT call this function outside of a guaranteed web window environment.
- * @param {Object} args - The args
- * @param {string} args.url - The URL to send to
- * @returns {HTMLImageElement}
+ * Send via fetch with keepalive true
+ * @param {Object} args - The args.
+ * @param {string} args.url - The URL to send to.
+ * @param {string=} args.body - The Stringified body.
+ * @param {string=} [args.method=POST] - The XHR method to use.
+ * @param {{key: string, value: string}[]} [args.headers] - The headers to attach.
+ * @returns {XMLHttpRequest}
  */
-submitData.img = function img ({ url }) {
-  var element = new Image()
-  element.src = url
-  return element
+export function fetchKeepAlive ({ url, body = null, method = 'POST', headers = [{ key: 'content-type', value: 'text/plain' }] }) {
```

Comment:
Does this cause any errors in FF or just not honor the "keep-alive" part?

Reply:
I will have to check but was that an issue the last time we tried fetch with keepalive?

Reply:
I don't believe we've ever actually utilized fetch keepalive in the agent, so i'm not sure what firefox foes there.  I'm assuming it just ignores that property and treats it like a standard fetch

Reply:
Tested in Chrome 65, edge 13, firefox 55...they all still make the fetch call and seem to ignore the keepalive part.